sessionctx/stmtctx/stmtctx.go

```diff
@@ -548,7 +548,11 @@ func (sc *StatementContext) CopTasksDetails() *CopTasksDetails {
 		sleepTime time.Duration
 		times     int
 	}
-	backoffInfo := make(map[string][]backoffItem)
+	backoffNum := 0
```

Comment:
The number of backoff types is limited, the map would be small enough.

Reply:
Addressed